README.md

```diff
@@ -106,7 +106,7 @@ Note: the instructions below are for Timescale's standard compute instance. For
 
 To enable pgvectorscale:
 
-1. Create a new [Timescale Service](https://console.cloud.timescale.com/dashboard/create_services).
+1. Create a new [Timescale account](https://console.cloud.timescale.com/signup?utm_campaign=vectorlaunch).
```

Comment:
Can we go with service please. This keeps us aligned with the docs: https://docs.timescale.com/getting-started/latest/services/.